### PR TITLE
Adding US egg production data for 2023-04-11

### PR DESCRIPTION
**Pull request by tracykteal**
Dataset from [US Egg Production Dataset](https://thehumaneleague.org/article/E008R01-us-egg-production-data) by @samaramendez.

### CODE REVIEW

File: data/2023/2023-04-11/readme.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+tuesdata <- tidytuesdayR::tt_load('2023-04-11')
+tuesdata <- tidytuesdayR::tt_load(2023, week = 15)
+
+eggproduction <- tuesdata$eggproduction
+cagefreepercentages <- tuesdata$cagefreepercentages
+
+
+# Or read in the data manually
+
+transitions <- readr::read_csv('https://raw.githubusercontent.com/rfordatascience/tidytuesday/master/data/2023/2023-04-11/egg-production.csv')
```
Comment (jonthegeek) [inline data/2023/2023-04-11/readme.md:96]:
```suggestion
eggproduction  <- readr::read_csv('https://raw.githubusercontent.com/rfordatascience/tidytuesday/master/data/2023/2023-04-11/egg-production.csv')
```

File: data/2023/2023-04-11/readme.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+timezones <- readr::read_csv('https://raw.githubusercontent.com/rfordatascience/tidytuesday/master/data/2023/2023-04-11/cage-free-percentages.csv')
```
Comment (jonthegeek) [inline data/2023/2023-04-11/readme.md:97]:
```suggestion
cagefreepercentages <- readr::read_csv('https://raw.githubusercontent.com/rfordatascience/tidytuesday/master/data/2023/2023-04-11/cage-free-percentages.csv')
```

**Review by jonthegeek — CHANGES_REQUESTED** [review]
Just a couple variable names to tweak in the data load, I'll commit these suggestions then merge!

**Review by jonthegeek — APPROVED** [review]
Ok, fixed!

**Comment by samaramendez** [comment]
Thanks for using my dataset! I've just updated the files in [the repository ](https://osf.io/z2gxn/) with the most recent data, if you'd like to include 2023 data.

**Comment by tracykteal** [comment]
That's great, thank you @samaramendez! We keep the datasets small for TidyTuesday, but I'll update the README to show that the most recent data is available in the repo.

**Pull request merged**